core/jazz_t-vault/components/utils/vault.js

```diff
@@ -124,7 +125,8 @@ function deleteSafe(safename, configData, vaultToken, onComplete) {
     logger.debug("deleteSafe response: " + JSON.stringify(response));
     if (response.statusCode && response.statusCode === 200) {
       logger.debug("Successfully deleted safe details: " + JSON.stringify(body));
-      return onComplete(null, body);
+      const message = { "message": `Safe ${safename.toLowerCase()} deleted successfully.` };
```

Comment:
Let us not change the case here. Also for the error messages, are we getting the input. If not, it might make sense to also make the error messages more friendly with some details similar to the success messages.